src/phpDebug.ts

```diff
@@ -365,6 +365,7 @@ class PhpDebugSession extends vscode.DebugSession {
                 exceptionText = response.exception.name + ': ' + response.exception.message // this seems to be ignored currently by VS Code
             } else if (this._args.stopOnEntry) {
                 stoppedEventReason = 'entry'
+                this._args.stopOnEntry = false
```

Comment:
`args` is documented as "The arguments that were given to launchRequest", mutating these at runtime would be confusing if in the future any other part of the code needs to access this and the values don't match the arguments that were given to launchRequest. It would be better to add a separate boolean flag like `hasStoppedOnEntry` that is set to `true` after the first time and also checked.

Reply:
I have introduced a flag in PhpDebugSession called _hasStoppedOnEntry. It is reset each time before calling the setpInto request.
Now that I think about it more, perhaps it would be better to store this flag into Connection class, as this is a connection specific state?